WeatherToday/Configurations/Configuration.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Configuration.swift
+//  WeatherToday
+//
+//  Created by Taher on 10/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+enum Configuration: String {
+
+    case debugStaging
+    case debugProduction
+    case releaseStaging
+    case releaseProduction
+
+    // MARK: - Current Configuration
+
+    static let current: Configuration = {
+        #if DEBUG && PRODUCTION
+        return .debugProduction
+        #elseif DEBUG && STAGING
+        return .debugStaging
+        #elseif RELEASE && STAGING
+        return .releaseStaging
+        #elseif RELEASE && PRODUCTION
+        return .releaseProduction
+        #endif
+    }()
+
+    static var baseWeatherURL: String {
+        guard let urlValue = Bundle.main.infoDictionary?["WEATHER_API_BASE_URL"] as? String else {
+            fatalError("No URL Found")
+        }
+        return "https://\(urlValue)/"
+    }
+
+    static var weatherApiKey: String {
+        switch current {
+        case .debugProduction:
```

Comment:
I think you can write this section like this, then later on if you need to add a new key you can move the case to a new block
```
static var weatherApiKey: String {
         switch current {
         case .debugProduction,
                  .releaseProduction, 
                  .debugStaging, 
                  .releaseStaging:           
             return "34e43ac725f3679763f219f9d2de0801"
         }
     }
```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/802414af6c444cf0ee7b0279d2ce9ceebce20e6d)